### ISSUE TITLE
How is BatchNorm implemented in CNNs?

### ISSUE THREAD

**Issue by Jzjerry** [alexmr09/Mixed-precision-Neural-Networks-on-RISC-V-Cores#3]
Hi,

I didn't find any code related to batch norm layers in the inference code, and I'd like to know how they are handled. Are they fused with Conv2D layers?

**Comment by alexmr09** [comment]
Currently, the batch normalization layers are fused with the convolutional layers before proceeding with fine-tuning the model's parameters.

**Comment by Jzjerry** [comment]
Got it, thanks!
